python/cli/prelude_cli/views/scm.py

```diff
@@ -355,6 +355,129 @@ def parse_from_partner_advisory(controller, partner, advisory_id):
             partner=Control[partner], advisory_id=advisory_id
         )
 
+@scm.command("list-policy-exceptions")
+@click.pass_obj
+@pretty_print
+def list_policy_exceptions(controller):
+    """List all policy exceptions"""
+    with Spinner(description="Querying Policy exceptions"):
+        return controller.list_policy_exceptions()
+
+@scm.command("update-policy-exception")
+@click.argument(
+    "partner",
+    type=click.Choice(
+        [c.name for c in Control if c != Control.INVALID], case_sensitive=False
+    ),
+)
+@click.option("-e", "--expires", help="Expiry Date (YYYY-MM-DD hh:mm:ss ([+-]hh:mm))", default=None, type=str)
+@click.option("-i", "--instance_id", required=True, help="instance ID of the partner")
+@click.option("-p", "--policy_id", required=True, help="ID of the policy to update")
+@click.option("-s", "--settings", required=True, help="Comma separated list of all setting names to be excluded")
+@click.pass_obj
+@pretty_print
+def update_policy_exception(controller, partner, expires, instance_id, policy_id, settings):
+    """Update policy exception"""
+    with Spinner(description=f"Updating Policy exception"):
+        return controller.put_policy_exceptions(
+            partner=Control[partner],
+            expires=expires,
+            instance_id=instance_id,
+            policy_id=policy_id,
+            setting_names=settings.split(",") if settings else None
+        )
+
+@scm.command("create-policy-exception")
+@click.argument(
+    "partner",
+    type=click.Choice(
+        [c.name for c in Control if c != Control.INVALID], case_sensitive=False
+    ),
+)
+@click.option("-e", "--expires", help="Expiry Date (YYYY-MM-DD hh:mm:ss ([+-]hh:mm))", default=None, type=str)
+@click.option("-i", "--instance_id", required=True, help="instance ID of the partner")
+@click.option("-p", "--policy_id", required=True, help="ID of the policy to create")
+@click.option("-s", "--settings", required=True, help="Comma separated list of all setting names to be excluded")
+@click.pass_obj
+@pretty_print
+def create_policy_exception(controller, partner, expires, instance_id, policy_id, settings ):
```

Comment:
extra space still on the rest of these :( 

we use the Black code formatter if you have it